contracts/deploy/01-outbox/01-arb-to-gnosis-outbox.ts

```diff
@@ -36,17 +36,17 @@ const paramsByChainId = {
     sequencerLimit: 86400, // 24 hours
   },
   HARDHAT: {
-    deposit: parseEther("5"), // 120 xDAI budget for timeout
+    deposit: parseEther("10"), //
```

Comment:
Nice, no comment leaves no chance to discrepancies! That is even better than the update I was expecting. It looks like the comment slashes where left though.

Reply:
I am also not sure what did the 120 xDai comment meant, its not a direct conversion for sure as 5 eth was 120 in mid 2017 ,maybe JB can help with it